eng/Signing.props

```diff
@@ -5,6 +5,14 @@
       TODO: Make SIGN001 suppressable.
     -->
     <FileSignInfo Include="Microsoft.VisualStudio.OLE.Interop.dll" CertificateName="None" />
+    <FileSignInfo Include="Microsoft.AspNetCore.ApiVersioning.dll" CertificateName="None" />
```

Comment:
Why is this now necessary?

Reply:
I updated the repo to latest SignTool and now the SIGN001 warnings are gone. Validated on this build: https://dnceng.visualstudio.com/internal/_build/results?buildId=128710